bitbots_vision/launch/vision_startup.launch

```diff
@@ -7,10 +7,11 @@
     <arg name="debug" default="false" doc="true: activates publishing of several debug images" />
     <arg name="use_game_settings" default="false" doc="true: loads additional game settings" />
 
-
+    <arg if="$(optenv IS_ROBOT false)" name="taskset" default="taskset -c 6,7"/>
```

Comment:
Does the env variable `IS_ROBOT` exist at the moment? Because it would be also possible to check `ROBOT_NAME` variable which also only exists if there is a robot.

Reply:
We added `IS_ROBOT` to the Ansible. We also tried to use `ROBOT_NAME`, but the `if` in launch files requires a bool, a string gives a type error and is not casted to true.

Reply:
Ah okay